Gemfile.lock

```diff
@@ -50,8 +50,6 @@ GEM
     airtable (0.0.9)
       httparty
     arel (7.1.4)
-    arel-helpers (2.6.1)
```

Comment:
Try re-running `make bundle` to see if that resolves your gemfile.lock issues.

If not, try to rebase on master, then `make bundle`.

Reply:
hmm it doesn't resolve the conflict issues let me just try to make another branch and do all the proper changes together